src/cpp/r/R/Api.R

```diff
@@ -706,12 +706,21 @@ options(terminal.manager = list(terminalActivate = .rs.api.terminalActivate,
 })
 
 .rs.addApiFunction("getThemeInfo", function() {
-   editor <- .Call("rs_readUiPref", "theme")
-   global <- .Call("rs_readUiPref", "flat_theme")
-
+   
+   # read theme preferences
+   editor <- .rs.readUiPref("theme")
+   global <- .rs.readUiPref("flat_theme")
+   
+   # if 'theme' is not set, try pulling from 'rstheme'
    if (is.null(editor)) {
-      editor <- "TextMate"
+      rstheme <- .rs.readUiPref("rstheme")
```

Comment:
`theme` is deprecated, it should always be `rstheme` now. Good catch.

Reply:
I wonder if we should instead read `rstheme` before `theme` then? Could these variables get out of sync?

Reply:
`theme` is unused everywhere else, we can just stop reading it altogether. For 1.2 people will only have a `theme` value in their prefs file if they upgraded from an older version. Unfortunately this means users will have to re-set their theme preference after upgrade, but the storage structure changed so we really had to do this.

Reply:
Got it. I'll remove the old `"theme"` usage altogether.